english/BayesTheorem/priors/priors.md

```diff
@@ -1 +1,3 @@
-Don't you dare ask me such things, son!
\ No newline at end of file
+<<<<<<< HEAD
+Don't you dare ask me such things, son! They come from expert knowledge; a meta-analysis of previous experiments; or, if you are an empirical 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.